src/main/java/org/embulk/parser/msgpack/MsgpackParserPlugin.java

```diff
@@ -627,6 +634,73 @@ public int compare(MessageBuffer o1, MessageBuffer o2)
         }
     }
 
+    private static Value repackValueInLongRange(final Value value) {
```

Comment:
A comment(why we need to repack Value) may help.
https://github.com/embulk/embulk-parser-msgpack/pull/27#issuecomment-1767658946

Reply:
Yeah, thought about that, but they will go away immediately very soon in v0.6.0 when moving to `JsonValue`. They're only for this version v0.5.0.